packages/sources/src/components/autosuggest/autosuggest.stories.ts

```diff
@@ -50,10 +53,27 @@ export function storiesOfAutosuggest<TemplateFnReturnType>(
     (a: Args) => {
       const args = a as AutosuggestArgs;
 
-      return autosuggestDemoTemplate(fetchSuggestions, args.onSelect, args.onChange, args.onSearch, args.suggestOnFocus, args.loading, args.loadingLabel, args.notFoundLabel);
+      return autosuggestDemoTemplate(fetchSuggestions, args.onSelect, args.onChange, args.onSearch, args.suggestOnFocus, args.loading, args.loadingLabel, args.loadingDelayed, args.notFoundLabel);
     }
   );
 
+  if (autosuggestLoadingDelayedTemplate) {
```

Comment:
Ik denk dat we _toch_ zonder deze template kunnen. Als we in de .spec navigeren naar een story met `&args=loadingDelayed:1000` kunnen we de loading delayed via storyBook args zetten, en daarmee ook de `autosuggestConnector` loadingDelayed vanuit de test bedienen.